common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -431,24 +443,11 @@ describe 'Problem', ->
         expect($('#answer_1_1')).toHaveHtml 'One'
         expect($('#answer_1_2')).toHaveHtml 'Two'
 
-      it 'toggle the show answer button', ->
-        spyOn($, 'postWithPrefix').and.callFake (url, callback) -> callback(answers: {})
-        @problem.show()
-        expect($('.show .show-label')).toHaveText 'Hide Answer'
-        expect(window.SR.readElts).toHaveBeenCalled()
-
-      it 'toggle the show answer button, answers are strings', ->
```

Comment:
Sorry, I did not intend that you delete the entire test point, just that you change the description message so that it does not reference "toggle the show answer button". We don't want to lose test coverage of window.SR.readElts.

So this test point should look like:

````
it 'sends the answers to the SR element', ->
    spyOn($, 'postWithPrefix').and.callFake (url, callback) -> callback(answers: '1_1': 'One', '1_2': 'Two')
    @problem.show()
    expect(window.SR.readElts).toHaveBeenCalledWith ['<p>Answer: One</p>', '<p>Answer: Two</p>']